src/NewProject/ProjectManager/project_manager.cpp

```diff
@@ -1796,6 +1807,9 @@ int ProjectManager::CreateAndAddFile(const QString& suffix,
     ret = CreateSDCFile(filename);
   } else if (!suffix.compare("pin", Qt::CaseInsensitive)) {
     ret = CreateSDCFile(filename);
+  } else if (suffix.compare("cpp", Qt::CaseInsensitive) == 0 ||
+             suffix.compare("c", Qt::CaseInsensitive) == 0) {
```

Comment:
Why "c" here ? Don't we want to distinguish them ?

Reply:
In this case it doesn't matter.